### PR TITLE
Fl 717 upgrade m UI

### PR DESCRIPTION
**Pull request by jennifer-fintlabs**
Upgraded MUI

### CODE REVIEW

**Comment by trondsevre** [comment]
Her har jeg ikke mye å bidra med. Dette får dere frontenderne ta dere av :)

File: src/common/button/AddButton.jsx [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-import Tooltip from "@material-ui/core/Tooltip";
-import { IconButton, withStyles } from "@material-ui/core";
-import AddIcon from "../../../node_modules/@material-ui/icons/CheckBoxOutlineBlank";
+import { Tooltip, IconButton } from '@mui/material';
 
-const styles = theme => ({
-  addIcon: {
+import AddIcon from "@mui/icons-material/CheckBoxOutlineBlank";
+
+const PREFIX = 'AddButton';
```
Comment (ingrie) [inline src/common/button/AddButton.jsx:8]: Why separate variable for prefix?
why not addIcon: "AddButton-addIcon"

File: src/common/button/AddButton.jsx [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+
+const classes = {
+  addIcon: `${PREFIX}-addIcon`
+};
+
+const StyledTooltip = styled(Tooltip)((
```
Comment (ingrie) [inline src/common/button/AddButton.jsx:14]: condense to one line? const StyledTooltip = styled(Tooltip)(({theme}) => ({

File: src/common/button/RemoveButton.jsx [hunk 3]
```diff
@@ -1,26 +1,40 @@
...
-import Tooltip from "@material-ui/core/Tooltip";
-import { IconButton, withStyles } from "@material-ui/core";
-import RemoveIcon from "../../../node_modules/@material-ui/icons/CheckBox";
+import { IconButton, Tooltip } from "@mui/material";
 
-const styles = theme => ({
-  addIcon: {
+import RemoveIcon from "@mui/icons-material/CheckBox";
+
+const PREFIX = 'RemoveButton';
```
Comment (ingrie) [inline src/common/button/RemoveButton.jsx:8]: same question about the prefix

File: src/common/button/RemoveButton.jsx [hunk 4]
```diff
@@ -1,26 +1,40 @@
...
+  [`& .${classes.removeIcon}`]: {
     color: theme.palette.secondary.main
   }
-});
+}));
+
 class RemoveButton extends Component {
   render() {
-    const { classes } = this.props;
+    const { } = this.props;
```
Comment (ingrie) [inline src/common/button/RemoveButton.jsx:31]: is this line necessary?

File: src/common/input-validation/AssetNameValidationInput.jsx [hunk 5]
```diff
@@ -35,7 +34,8 @@ class AssetNameValidationInput extends React.Component {
     const { name, title, className } = this.props;
     return (
       <FormControl
-        fullWidth
+        // fullWidth
```
Comment (ingrie) [inline src/common/input-validation/AssetNameValidationInput.jsx:37]: better to remove if it's not a temporary commented out line

File: src/common/input-validation/PackageIdValidationInput.jsx [hunk 6]
```diff
@@ -1,24 +1,44 @@
...
+
+const classes = {
+    realm: `${PREFIX}-realm`,
+    label: `${PREFIX}-label`,
+    input: `${PREFIX}-input`
+};
+
+const StyledFormControl = styled(FormControl)((
+    {
+        theme
```
Comment (ingrie) [inline src/common/input-validation/PackageIdValidationInput.jsx:23]: seeing this over 3 lines a lot. is this automatically formatted? takes up a lot of space 😛

File: src/common/message-box/InformationMessageBox.jsx [hunk 7]
```diff
@@ -45,9 +57,9 @@ class WarningMessageBox extends React.Component {
   }
 
   render() {
-    const { classes } = this.props;
+    const { } = this.props;
```
Comment (ingrie) [inline src/common/message-box/InformationMessageBox.jsx:60]: redundant?

File: src/features/access-package/edit/EntityTable.jsx [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import TableCell from "@material-ui/core/TableCell";
-import {Checkbox, TableRow} from "@material-ui/core";
+import TableCell from '@mui/material/TableCell';
+import {Checkbox, TableRow} from "@mui/material";
```
Comment (ingrie) [inline src/features/access-package/edit/EntityTable.jsx:3]: more consistent use of " vs '

File: src/features/access-package/edit/EntityTableHeader.jsx [hunk 9]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import TableCell from "@material-ui/core/TableCell";
-import {TableRow} from "@material-ui/core";
-import ModifyIcon from "@material-ui/icons/Edit";
-import BulkIcon from "@material-ui/icons/FormatListBulleted";
-import SingleIcon from "@material-ui/icons/LooksOne";
-import Tooltip from "@material-ui/core/Tooltip";
+import { TableCell, TableRow, Tooltip } from '@mui/material';
```
Comment (ingrie) [inline src/features/access-package/edit/EntityTableHeader.jsx:2]:
```suggestion
import { TableCell, TableRow, Tooltip } from "@mui/material";
```

File: src/features/access-package/edit/SelectAllEntitiesCheckboxes.jsx [hunk 10]
```diff
@@ -1,22 +1,31 @@
 import React from 'react';
-import TableCell from "@material-ui/core/TableCell";
-import {Checkbox, TableRow, Typography} from "@material-ui/core";
-import {makeStyles} from "@material-ui/core/styles";
+import { styled } from '@mui/material/styles';
```
Comment (ingrie) [inline src/features/access-package/edit/SelectAllEntitiesCheckboxes.jsx:2]:
```suggestion
import { styled } from "@mui/material/styles";
```

File: src/features/access-package/edit/SelectAllEntitiesCheckboxes.jsx [hunk 11]
```diff
@@ -1,22 +1,31 @@
...
+import {Checkbox, TableRow, Typography, TableCell} from "@mui/material";
+const PREFIX = 'SelectAllEntitiesCheckboxes';
```
Comment (ingrie) [inline src/features/access-package/edit/SelectAllEntitiesCheckboxes.jsx:4]:
```suggestion
const PREFIX = "SelectAllEntitiesCheckboxes";
```

File: src/features/asset/view/AssetTabView.jsx [hunk 12]
```diff
@@ -46,10 +57,10 @@ class AssetTabView extends React.Component {
   }
 
   render() {
-    const { classes, theme } = this.props;
+    // const { classes, theme } = this.props;
```
Comment (ingrie) [inline src/features/asset/view/AssetTabView.jsx:60]: remove unused comments

File: src/features/basic-test/BasicTestRunButton.jsx [hunk 13]
```diff
@@ -52,13 +68,13 @@ class BasicTestRunButton extends React.Component {
 
   render() {
     const { loading, success } = this.props;
-    const { classes } = this.props;
+    const { } = this.props;
```
Comment (ingrie) [inline src/features/basic-test/BasicTestRunButton.jsx:71]: I think this is redundant

File: src/features/clients/view/AddAccessPackageToClient.jsx [hunk 14]
```diff
@@ -1,5 +1,5 @@
 import React, {useContext, useState} from "react";
-import {List} from "@material-ui/core";
+import { List } from '@mui/material';
```
Comment (ingrie) [inline src/features/clients/view/AddAccessPackageToClient.jsx:2]: consistent use of space or no space in brackets

File: src/features/component/ComponentContainer.jsx [hunk 15]
```diff
@@ -49,10 +55,10 @@ class ComponentContainer extends React.Component {
   }
 
   renderPosts() {
-    const { classes, components } = this.props;
+    const {  components } = this.props;
     //const components = this.getComponentsList();
```
Comment (ingrie) [inline src/features/component/ComponentContainer.jsx:59]: remove unused comments?

File: src/features/nogo/NoGoMessage.jsx [hunk 16]
```diff
@@ -1,24 +1,34 @@
...
         //height: "25%",
         width: 178
     }
-});
+}));
 
 class NoGoMessage extends Component {
     render() {
-        const {classes} = this.props;
+        // const {classes} = this.props;
```
Comment (ingrie) [inline src/features/nogo/NoGoMessage.jsx:29]: remove unused comment

**Review by ingrie — COMMENTED** [review]
All in all, not a lot to comment on, general nit picking, like consistent use of single or regular quotes ' or ", spacing within imports in brackets and some comments that can be removed. 
Also the empty { } = this.props which is found a few places I think(!) is redundant.
Left some comments to show examples 😊

**Pull request closed without merging**